plugins/modules/clustering/znode.py

```diff
@@ -48,6 +48,11 @@
             - Recursively delete node and all its children.
         type: bool
         default: 'no'
+    usessl:
```

Comment:
I would call this
```suggestion
    use_tls:
```
since the protocol hasn't been called SSL for decades.